applications/models.py

```diff
@@ -167,10 +168,14 @@ def complete(self):
         ],
         target=AppStates.AWAITING_USER_SUBMISSIONS.value,
     )
-    def upload_resume(self, resume_file):
+    def add_resume(self, *, resume_file=None, linkedin_url=None):
         """Save resume and make sure that the state can be transitioned to a new state"""
-        validate_file_extension(resume_file)
-        self.resume_file = resume_file
+        if resume_file:
```

Comment:
Please update the docstring: `Add resume and/or linkedin URL to the application`